components/ruby/lib/chef-licensing/license_key_fetcher.rb

```diff
@@ -71,7 +71,8 @@ def perform_on_prem_operations
 
       unless @license_keys.empty?
         # Licenses expiration check
-        if licenses_active?
+        # Client API possible errors will be handled in software entitlement check call (made after this)
+        if licenses_active? || client_api_call_error
```

Comment:
makes sense. Maybe add a comment that this client_api_call_error is set during the `licenses_active?` check? Either way it works.

Reply:
Done